clang-tools-extra/clang-reorder-fields/ReorderFieldsAction.cpp

```diff
@@ -118,47 +118,19 @@ findMembersUsedInInitExpr(const CXXCtorInitializer *Initializer,
   return Results;
 }
 
-/// Returns the next token after `Loc` (including comment tokens).
-static std::optional<Token> getTokenAfter(SourceLocation Loc,
```

Comment:
This looks like just a better formatted `findNextTokenIncludingComments`

Reply:
Yes, that's why we wanted to get rid of that code, it does not really make sense to have 3 quasi-identical copies of the same code.